src/checkpointing/checkpoint.py

```diff
@@ -0,0 +1,32 @@
+import sys
+from datetime import datetime
+
+class Checkpoint:
+    def __init__(self):
+        self._file = "checkpoint.txt"
```

Comment:
How do you feel about throwing the checkpoint file into assets/ ?

Reply:
Ah yeah didn't think of that, that does seem like a better location for it. You can put the file path within the quotes and it will direct it there correct?

Reply:
I'm pretty sure that it uses the directory the method is called from as the working directory, so if it's used in main.py, it should use the top directory of our project. That said, you could probably just run it in several places and see how it acts.